server.go

```diff
@@ -2063,6 +2072,14 @@ func (s *server) Start() error {
 			cleanup = cleanup.add(s.torController.Stop)
 		}
 
+		if s.kvdbPeerStorage != nil {
+			if err := s.kvdbPeerStorage.Start(); err != nil {
+				startErr = err
+				return
+			}
+			cleanup = cleanup.add(s.kvdbPeerStorage.Stop)
+		}
```

Comment:
You should omit this block completely. We already take care of this in the backend initialization.